application/F3DStarter.cxx

```diff
@@ -1636,4 +1639,22 @@ void F3DStarter::AddCommands()
         this->LoadFileGroup(index);
       }
     });
+    interactor.addCommand("open_file", 
```

Comment:
look at `add_files` commnad. Your `open_files` should basically do the same after selecting files in the dialog

Reply:
done, I originally found the engine commands when I was first reading through the project so I was using the wrong api. Should be fixed, filename working as expected.